### PR TITLE
QA scripts

### PR DESCRIPTION
**Pull request by NeoTheThird**
This adds basic documentation for the QA scripts we are shipping in OTA-4, see ubports/ubuntu-touch#688.

### CODE REVIEW

File: about/process/ppa.rst [hunk 1]
```diff
@@ -1,8 +1,16 @@
...
 ================
 
-This is the package archive system for UBports projects. It hosts various PPAs containing all deb-components of Ubuntu Touch. New PPAs can be created dynamically by the CI server using a special :doc:`git-branch naming convention <branch-naming>`.
+This is the package archive system for UBports projects. It hosts various PPAs containing all deb-components of Ubuntu Touch.
 
-The name of the branch translates literally to the name of the PPA: ``http://repo.ubports.com/dists/[branch name]``
+Package-naming convention
+-------------------------
+
+The name of the deb-package and the name of the git-repository should be the same.
```
Comment (UniversalSuperBox) [inline about/process/ppa.rst]: This isn't exactly true. Our repositories are named by the name of the *source* package. In some cases, they have the `-packaging` moniker to denote that the actual upstream source is retrieved from somewhere else.

File: contribute/quality-assurance.rst [hunk 2]
```diff
@@ -20,7 +20,9 @@ If you find two issues describing the same problem, leave a comment and try to f
...
+Pull-requests can be tested using the `QA scripts <https://github.com/ubports/ubports-qa-scripts>`_. Run ``ubports-qa -h`` for usage information.
+
+Pull-request has been accepted, the issue is moved to the quality assurance column of the `GitHub project <https://github.com/ubports/ubuntu-touch/projects/3>`__. Check if the issue is still present in the latest update on the devel channel and try to find any problems it is causing. Check if the developer mentioned specific things to look out for when testing and leave a comment detailing your experience. If you have write-access to the repository, you can move the issue back to **In Development** or forward to **Release Candidate** as specified by the :doc:`issue tracking guidelines </about/process/issue-tracking>`.
```
Comment (UniversalSuperBox) [inline contribute/quality-assurance.rst]: The first sentence on this line is a fragment.

**Comment by UniversalSuperBox** [comment]
Since this includes information about Debian packaging that I'm not so sure of, I'd like @tsimonq2 to review it as well. I'll keep my blocking review until that has been finished.

**Comment by tsimonq2** [comment]
Looks good to me.

**Pull request merged**